Reductions (intro) - Copyedits (#210)
mlsa-book/MLSA: book/P4C19_reductions.qmd

# Reduction Techniques for Survival Analysis {#sec-reduction-techniques}

This part of the book introduces and formalizes the concept of *reduction* for survival analysis\index{reduction}. 
A reduction is defined as "a complex problem decomposed into simpler subproblems so that a solution to the subproblems gives a solution to the complex problem" [@beygelzimerLearningReductionsThat2016].

> Reduction techniques can simplify the application of machine learning methods to survival analysis in two ways: by transforming a survival task to a more standard regression or classification task, allowing use of standard machine learning methods, and by transforming a more complex event history analysis task into a set of single-event tasks, allowing use of single-event machine learning methods. As we will see, the two reduction paradigms can also be combined.

Author: ```suggestion
Reduction techniques can simplify the application of machine learning methods to survival analysis in two ways: by transforming a survival task to a regression or classification task, allowing use of standard machine learning methods; and by transforming an event history analysis task into a set of single-event tasks, allowing the use of single-event machine learning methods.
As will be seen in the upcoming chapters, the two reduction paradigms can also be combined.
```
